modules/openid_auth/app/controllers/openid_auth/v2/validation_controller.rb

```diff
@@ -81,6 +86,28 @@ def setup_structure
         end
         payload_object
       end
+
+      def charon_token_screen?(payload_object)
+        act_vid = payload_object.act[:vista_id]
+        sta3n = payload_object.launch['sta3n']
+        return false unless !act_vid.nil? && !sta3n.nil?
```

Comment:
There has to be launch scope for those token issued for certain particular audiences. because the launch context is where the station id (used as a charon parameter) comes from.  Not all ssoi tokens will need additional validation. But any ssoi token with an audience flagged as one that should need additional charon validation is not authorized if it doesn't have a launch context.

Reply:
Ok so in that case the response will be as follows:

```json
{"errors":[{"title":"Not authorized","detail":"Invalid request","code":"401","status":"401"}]}
```

By saying "not all ssoi tokens will need additional validation" you are saying they will not hit the validation endpoint?

Reply:
no. but rather not go through additional checks, using charon during validation.